geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/RegionManagementRestSecurityDUnitTest.java

```diff
@@ -0,0 +1,155 @@
+    regionConfig.setName("customers");
+    regionConfig.setType("REPLICATE");
+    ObjectMapper mapper = new ObjectMapper();
+    String json = mapper.writeValueAsString(regionConfig);
+
+    ClusterManagementResult result =
+        restClient.doPostAndAssert("/regions", json)
+            .hasStatusCode(401)
+            .getClusterManagementResult();
+
+    assertThat(result.isRealizedOnAllOrNone()).isTrue();
+    assertThat(result.isPersisted()).isFalse();
+  }
+
+  @Test
+  public void createRegionWithBadCredentials_failsWithAuthenticationError() throws Exception {
+    RegionConfig regionConfig = new RegionConfig();
+    regionConfig.setName("customers");
+    regionConfig.setType("REPLICATE");
+    ObjectMapper mapper = new ObjectMapper();
+    String json = mapper.writeValueAsString(regionConfig);
+
+    ClusterManagementResult result =
+        restClient.doPostAndAssert("/regions", json, "baduser", "badpassword")
+            .hasStatusCode(401)
+            .getClusterManagementResult();
+
+    assertThat(result.isRealizedOnAllOrNone()).isTrue();
+    assertThat(result.isPersisted()).isFalse();
+  }
+
+  @Test
+  public void createRegionNotAuthorized_failsWithAuthorizationError() throws Exception {
+    RegionConfig regionConfig = new RegionConfig();
+    regionConfig.setName("customers");
+    regionConfig.setType("REPLICATE");
+    ObjectMapper mapper = new ObjectMapper();
+    String json = mapper.writeValueAsString(regionConfig);
+
+    ClusterManagementResult result =
+        restClient.doPostAndAssert("/regions", json, "notauthorized", "notauthorized")
+            .hasStatusCode(403)
+            .getClusterManagementResult();
+
+    assertThat(result.isRealizedOnAllOrNone()).isTrue();
+    assertThat(result.isPersisted()).isFalse();
+  }
+
+  @Test
+  public void createRegionWithCredentials_CreatesRegion() throws Exception {
+    RegionConfig regionConfig = new RegionConfig();
+    regionConfig.setName("customers");
+    regionConfig.setType("REPLICATE");
+    ObjectMapper mapper = new ObjectMapper();
+    String json = mapper.writeValueAsString(regionConfig);
+
+    ClusterManagementResult result =
+        restClient.doPostAndAssert("/regions", json, "datamanage", "datamanage")
+            .hasStatusCode(201)
+            .getClusterManagementResult();
+
+    assertThat(result.isRealizedOnAllOrNone()).isTrue();
+    assertThat(result.isPersisted()).isTrue();
+    assertThat(result.getMemberStatuses()).containsKeys("server-1").hasSize(1);
+
+    // make sure region is created
+    server.invoke(() -> verifyRegionCreated("customers", "REPLICATE"));
+
+    // make sure region is persisted
+    locator.invoke(() -> verifyRegionPersisted("customers", "REPLICATE"));
+
+    // verify that additional server can be started with the cluster configuration
+    cluster.startServerVM(2, config, locator.getPort());
+  }
+
+
+  private static void verifyRegionPersisted(String regionName, String type) {
```

Comment:
since this test is very similar to the RegionManagementDunitTest, should we just add the security manager to that test and then it should cover all cases.

Reply:
Certainly could. I just figured on keeping both the secured and unsecured tests as separate entities.